pages/home/auth/secure-auth-production.mdx

```diff
@@ -0,0 +1,127 @@
+
+This setting is configured in the Arcade Dashboard.
+
+
+## Use the Arcade session verifier
+
+If you're building a proof-of-concept app or a solo project, use the Arcade session verifier. This option is on by default when you create a new Arcade.dev account.
+
+When you authorize a tool, you'll be prompted to sign in to your Arcade.dev account. If you are already signed in (to the Arcade Dashboard, for example), this verification will succeed silently.
+
+<Note>
+  Arcade's default OAuth apps can *only* be used with the Arcade session verifier.
+  If you are building a multi-user production app, you must obtain your own OAuth app credentials and add them to Arcade.
+  For example, [configure Google auth in the Arcade Dashboard](https://docs.arcade.dev/home/auth-providers/google#access-the-arcade-dashboard).
+</Note>
+
+The Arcade.dev session verifier helps keep your auth flows secure while you are building and testing your agent or app. When you're ready to share your work with others, implement a [custom session verifier](#set-up-a-custom-session-verifier) so your users don't need to sign in to Arcade.dev.
+
+
+## Set up a custom session verifier
+
+In a production application or agent, user sessions are verified by your code, not Arcade.dev. To enable this, build a custom verifier route and add the URL to the Arcade Dashboard.
+
+When your users authorize a tool, Arcade.dev will redirect the user's browser to your verifier route with some information in the query string. Your custom verifier route must send a response back to Arcade.dev to confirm the user's session details.
+
+If you need help, start a [GitHub discussion](https://github.com/ArcadeAI/arcade-ai/discussions) and we'll be happy to assist.
+
+import { Steps, Tabs } from "nextra/components";
+
+<Steps>
+
+### Build a new route
+
+Create a public route in your app or API that can accept a browser redirect (HTTP 303) from Arcade.dev after a user has authorized a tool.
+
+The route must gather the following information:
+
+- The `flow_id` from the current URL's query string
+- The unique ID of the user currently signed in. This varies depending on how your app is built, but it is typically retrieved from a session cookie or other secure storage. It **must** match the user ID that your code specified at the start of the authorization flow.
+
+
+### Verify the user's session details
+
+Use the Arcade SDK (or our REST API) to verify the user's session details.
+
+<Warning>
+  Because this request uses your Arcade API key, it *must not* be made from the client side (a browser or desktop app). This code must be run on a server.
+</Warning>
+
+<Tabs items={["REST"]} storageKey="preferredLanguage">
+<Tabs.Tab>
+```text
+POST https://cloud.arcade.dev/api/v1/oauth/confirm_user
+Authorization: Bearer <arcade_api_key>
+Content-Type: application/json
+
+{
+  "flow_id": "<flow_id from the query string>",
+  "user_id": "<the current user's ID>"
+}
+```
+</Tabs.Tab>
+</Tabs>
```

Comment:
We will get this into the client libs soon, yes?

Reply:
Yes, next on my list. That will result in more tabs here